docs/getting-started/installing/ha-master.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Highly available Kontena Master
+---
+
+# Running Kontena master in HA setup
+
+Kontena master can be run in HA setup where multiple instances of the Kontena master form one highly available logical master.
+
+## Requirements for HA
+
+- Master nodes:
+  - Any modern Linux distribution with support for Docker and docker-compose. CoreOS and Ubuntu are more heavily tested.
```

Comment:
There was an empty bulletpoint which made some oddness. Removing that and looks ok on `gitbook serve`